meson.build

```diff
@@ -1,4 +1,4 @@
-project('vkd3d-proton', ['c'], version : '2.6', meson_version : '>= 0.49', default_options : [
+project('vkd3d-proton', ['c'], version : '2.6', meson_version : '>= 0.55', default_options : [
```

Comment:
Is 0.55 required?

Reply:
Only for the patch_directory portion of the .wrap file. I will find a way around that.